codepipeline-tf/terraform.auto.tfvars

```diff
@@ -0,0 +1,44 @@
+##################################################
+# tfvars
+##################################################
+
+
+namespace_name = "example"
+account_id     = "your-account-id"
+domain_name    = "your-domain"
+
+s3_bucket_name = "terraform-state-xc3-example-pipeline"
+
+
+xc3_codepipeline_role  = "arn:aws:iam::201635854701:role/xccc-pipeline-role"
+codebuild_service_role = "arn:aws:iam::201635854701:role/service-role/codebuild-test-service-role"
+codestar_connections   = "arn:aws:codestar-connections:eu-west-1:201635854701:connection/68b68fa1-9687-405a-b75c-cbf1f5ca6de5"
```

Comment:
can we pass it via env or ssm parameter because it will be different for each repo you can mention it in pre-req thing

Reply:
Yes, you're right. but when we create connection with resource than we'll pass the arn of role that is created on runtime.